COPY/etc/httpd/conf.d/cfme-https-application.conf

```diff
@@ -38,8 +38,9 @@ SSLCertificateKeyFile /var/www/miq/vmdb/certs/server.cer.key
 # Redirect favicon.ico
 RewriteEngine On
 <ifmodule mod_rewrite.c>
-  RewriteCond %{REQUEST_URI} ^/favicon\.ico$ [NC]
-  RewriteRule (.*) /assets/favicon.ico       [L,R=301]
+  RewriteCond %{REQUEST_URI} ^/favicon\.ico$        [NC]
```

Comment:
Thanks @kbrock Yes, I just changed the spacing to line up the three entries. I'll need to update this PR anyway to handle additional URLs. Will do that shortly